### PR TITLE
Setup travis CI

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel mrunalp@gmail.com

### CODE REVIEW

**Comment by mrunalp** [comment]
@runcom PTAL

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: go
+go:
+  - 1.6.2
```
Comment (runcom) [inline .travis.yml]: is 1.6.3 already out?

**Comment by runcom** [comment]
LGTM (other than the silly question lol)

File: .travis.yml (hunk 1 above)
Comment (mrunalp) [inline .travis.yml]: Wasn't sure if it is available on travis. I'll check and update if required.

**Pull request merged**